### PR TITLE
Remove unused code, update the example and README

### PR DESCRIPTION
**Pull request by cnunciato**
This change includes some cleanup due to this code having been moved into pulumi/pulumi:

* Remove Dockerfile and entrypoint.sh
* Update the example workflow
* Update the README to direct

Signed-off-by: Christian Nunciato <c@nunciato.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,12 @@ Pulumi's GitHub Actions deploy apps and infrastructure to your cloud of choice,
 and GitHub. This includes previewing, validating, and collaborating on proposed deployments in the context of Pull
 Requests, and triggering deployments or promotions between different environments by merging or directly committing code.
 
+**Note**: This repository no longer contains the code responsible for creating the `pulumi/actions`
```
Comment (chrsmith) [inline README.md]: "This repository no longer contains the code responsible for creating the `pulumi/actions` Docker container." 

doesn't quite read right, especially since it implies that the user navigated to the wrong location.

Instead, I'd rephrase as:

> NOTE: This repository contains samples and additional documentation for using Pulumi's Github Actions container. The specific container definition and associate scripts are found
> \[here\].

File: examples/main.workflow [hunk 2]
```diff
@@ -1,13 +1,13 @@
-workflow "Update" {
+workflow "Update " {
```
Comment (chrsmith) [inline examples/main.workflow]: Nit: Why the space after "Update"?

Comment (cnunciato) [reply examples/main.workflow]: Mistake, I'll fix, thanks for noticing it.
